lib/downloadCargoPackageFromGit.nix

```diff
@@ -8,8 +9,6 @@ let
   inherit (pkgsBuildBuild)
     fetchgit
     stdenv;
-
-  craneUtils = pkgsBuildBuild.callPackage ../pkgs/crane-utils { };
```

Comment:
Oh this is a nice side benefit. When I spliced crane itself I didn't consider we could actually use this ourselves. I only did it cause it seemed the correct thing to do.

Reply:
Yeah this was a workaround to force the correct version to be used before splicing was fully sorted out, but now that you came along and fixed things end to end it was time to get rid of it :smile: